priv/repo/migrations/20231118160541_create_exam_and_student_exam_table.exs

```diff
@@ -0,0 +1,25 @@
+defmodule Dbservice.Repo.Migrations.CreateExamAndStudentExamTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:exam) do
+      add :name, :string
+      add :registration_deadline, :utc_datetime
+      add :date, :utc_datetime
+
+      timestamps()
+    end
+
+    create table(:student_exam_record) do
+      add :student_id, references(:student, on_delete: :nothing)
+      add :exam_id, references(:exam, on_delete: :nothing)
```

Comment:
dont u want to add an index? 

create index(:student_exam_record, [:student_id])
create index(:student_exam_record, [:exam_id])

or maybe composite index on [:student_id, :exam_id]